### PR TITLE
What's new in 8.16

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves https://github.com/elastic/security-docs/issues/5953.

Preview: [What’s new in 8.16](https://security-docs_bk_6077.docs-preview.app.elstc.co/guide/en/security/master/whats-new.html)

### CODE REVIEW

File: docs/whats-new.asciidoc [hunk 1]
```diff
@@ -14,143 +14,164 @@ Other versions: {security-guide-all}/8.14/whats-new.html[8.14] | {security-guide
...
 [role="screenshot"]
-image::whats-new/images/8.15/required-fields-related-integrations.png[The Required fields and Related integrations fields highlighted in the Create new rule UI]
+image::whats-new/images/8.16/install-enable-rules.png[Install and enable rules, 80%]
 
 [float]
-=== Suppress alerts for {ml} and {esql} rules
+=== Run rules manually
 
-{security-guide}/alert-suppression.html[Alert suppression] now supports the {ml} and {esql} rule types. You can use it to reduce the number of repeated or duplicate detection alerts generated from {ml} and {esql} rules.
+{security-guide}/rules-ui-management.html#manually-run-rules[Manually run rules] for testing purposes or additional rule coverage. Details about manual runs (such as the status of each run, the total number of runs that will occur, and more) are shown on the **Execution results** tab of the rule details page. Alerts generated from manual rule runs have the `kibana.alert.rule.execution.type: manual` field value pair in the alert document.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:85]: After looking at this again, we don't need to highlight that last detail. 

```suggestion
{security-guide}/rules-ui-management.html#manually-run-rules[Manually run rules] for testing purposes or additional rule coverage. Details about manual runs (such as the status of each run, the total number of runs that will occur, and more) are shown on the **Execution results** tab of the rule details page. 
```

**Review by nastasha-solomon — COMMENTED** [review]
Just left one comment. Everything else looks good from my end - thank you!

File: docs/whats-new.asciidoc [hunk 2]
```diff
@@ -14,143 +14,164 @@ Other versions: {security-guide-all}/8.14/whats-new.html[8.14] | {security-guide
...
-== Filter out process descendants
+=== New rules for {elastic-defend}'s endpoint protection features
 
-Create an {security-guide}/event-filters.html[event filter] that excludes the descendant events of a specific process, but still includes the primary process itself. This can help you limit the amount of events ingested into {elastic-sec}.
+New prebuilt rules tailored for each of {elastic-defend}'s endpoint protection features—malware, ransomware, memory threats, and malicious behavior—allow you to configure actions tailored for detection or prevention of each type.
+////
 
 [role="screenshot"]
-image::whats-new/images/8.15/event-filter-process-descendants.png[Add event filter flyout, 70%]
+image::whats-new/images/8.16/endpoint-protection-rules.png[Endpoint protection rules]
```
Comment (joepeeples) [inline docs/whats-new.asciidoc:156]: Need to comment out this image, too — it shows the new prebuilt rules for Defend.

Comment (natasha-moore-elastic) [reply docs/whats-new.asciidoc:154]: Good catch, thanks!

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**